### DISCUSSION TITLE
Healthcheck endpoint

### DISCUSSION THREAD

**Discussion by binaryfire** [outline/outline#5057] [Ideas]
It'd be great to have a `/health` endpoint so we can add automated healthchecks for Outline

**Comment by tommoor** [comment]
This exists as `/_health`

**Comment by binaryfire** [comment]
@tommoor Nice. 

Would it be possible to add curl to the Docker image? This would allow us to set a healthcheck for the Outline container in docker-compose.

**Reply by tommoor** [reply]
Done, will be in the next release:
https://github.com/outline/outline/commit/dafc4fb6090315cbcfb96d52345b1aa40495d79b
